django/contrib/admin/static/admin/css/base.css

```diff
@@ -87,7 +87,7 @@ html[data-theme="light"],
 }
 
 html, body {
-    height: 100%;
+    min-height: 100%;
```

Comment:
Article that you mentioned recommends using `100vh` not `100%`. (\cc @mgaligniana).